src/lib/Libifl/pbs_loadconf.c

```diff
@@ -906,7 +906,12 @@ __pbs_loadconf(int reload)
 		free(value);
 	}
 	if ((gvalue = getenv(PBS_CONF_ENCRYPT_METHOD)) != NULL) {
-		char *value = convert_string_to_lowercase(gvalue);
```

Comment:
@vchlum can we create small func to get value (while allowing for unsetting it) so we can use it of other pbs.conf variables like (PBS_AUTH_METHOD, PBS_SUPPORTED_AUTH_METHODS etc...)

Reply:
OK, I simplified it by creating the function ensure_string_not_null(). I don't know which other variables should be allowed to be unset. E.g. PBS_AUTH_METHOD or PBS_AUTH_METHOD should be IMO at least 'resvport'. Not empty.